diff -Nupr src.orig/mm/slub.c src/mm/slub.c
--- src.orig/mm/slub.c	2022-04-29 16:08:21.601781233 -0400
+++ src/mm/slub.c	2022-04-29 16:08:38.713842130 -0400
@@ -6107,6 +6107,9 @@ void get_slabinfo(struct kmem_cache *s,
 	int node;
 	struct kmem_cache_node *n;
 
+	if (!jiffies)
+		printk("slabinfo\n");
+
 	for_each_kmem_cache_node(s, node, n) {
 		nr_slabs += node_nr_slabs(n);
 		nr_objs += node_nr_objs(n);
