diff -Nupr src/mm/slub.c src/mm/slub.c
--- src/mm/slub.c	2020-03-11 11:23:32.406727384 +0000
+++ src/mm/slub.c	2020-03-11 11:28:27.973568215 +0000
@@ -5836,6 +5836,9 @@ void get_slabinfo(struct kmem_cache *s,
 	int node;
 	struct kmem_cache_node *n;
 
+	if (!jiffies)
+		printk("slabinfo\n");
+
 	for_each_kmem_cache_node(s, node, n) {
 		nr_slabs += node_nr_slabs(n);
 		nr_objs += node_nr_objs(n);
