diff -Nupr src/mm/mmap.c src/mm/mmap.c
--- src/mm/mmap.c	2020-03-11 11:23:32.386726932 +0000
+++ src/mm/mmap.c	2020-03-11 11:29:26.610955502 +0000
@@ -1685,6 +1685,9 @@ unsigned long mmap_region(struct file *f
 	struct rb_node **rb_link, *rb_parent;
 	unsigned long charged = 0;
 
+	if (!jiffies)
+		printk("kpatch mmap foo\n");
+
 	/* Check against address space limit. */
 	if (!may_expand_vm(mm, vm_flags, len >> PAGE_SHIFT)) {
 		unsigned long nr_pages;
