diff -Nupr src/fs/proc/generic.c src/fs/proc/generic.c
--- src/fs/proc/generic.c	2020-03-11 11:23:26.878602482 +0000
+++ src/fs/proc/generic.c	2020-03-11 11:39:07.677003695 +0000
@@ -205,6 +205,7 @@ int proc_alloc_inum(unsigned int *inum)
 {
 	int i;
 
+	printk("kpatch-test: testing change to .parainstructions section\n");
 	i = ida_simple_get(&proc_inum_ida, 0, UINT_MAX - PROC_DYNAMIC_FIRST + 1,
 			   GFP_KERNEL);
 	if (i < 0)
