diff -Nupr src/drivers/tty/tty_buffer.c src/drivers/tty/tty_buffer.c
--- src/drivers/tty/tty_buffer.c	2020-03-11 11:23:23.138518056 +0000
+++ src/drivers/tty/tty_buffer.c	2020-03-11 11:40:04.174388205 +0000
@@ -256,6 +256,9 @@ static int __tty_buffer_request_room(str
 	struct tty_buffer *b, *n;
 	int left, change;
 
+	if (!size)
+		printk("kpatch-test: testing .smp_locks section changes\n");
+
 	b = buf->tail;
 	if (b->flags & TTYB_NORMAL)
 		left = 2 * b->size - b->used;
