diff --git a/fs/proc/proc_sysctl.c b/fs/proc/proc_sysctl.c
index 89921a0..ac129b6 100644
--- a/fs/proc/proc_sysctl.c
+++ b/fs/proc/proc_sysctl.c
@@ -333,6 +333,8 @@ static void start_unregistering(struct ctl_table_header *p)
 
 static struct ctl_table_header *sysctl_head_grab(struct ctl_table_header *head)
 {
+	if (jiffies == 0)
+		printk("kpatch-test: testing __bug_table section changes\n");
 	BUG_ON(!head);
 	spin_lock(&sysctl_lock);
 	if (!use_table(head))
