diff --git a/fs/proc/meminfo.c b/fs/proc/meminfo.c
index 2fb0484..79ead86 100644
--- a/fs/proc/meminfo.c
+++ b/fs/proc/meminfo.c
@@ -30,6 +30,8 @@ static void show_val_kb(struct seq_file *m, const char *s, unsigned long num)
 	seq_write(m, " kB\n", 4);
 }
 
+static int foo = 5;
+
 static int meminfo_proc_show(struct seq_file *m, void *v)
 {
 	struct sysinfo i;
@@ -141,6 +143,7 @@ static int meminfo_proc_show(struct seq_file *m, void *v)
 	show_val_kb(m, "CmaFree:        ",
 		    global_zone_page_state(NR_FREE_CMA_PAGES));
 #endif
+	seq_printf(m, "kpatch:         %d\n", foo);
 
 	hugetlb_report_meminfo(m);
 
