diff --git a/fs/proc/meminfo.c b/fs/proc/meminfo.c
index 2fb0484..eb61884 100644
--- a/fs/proc/meminfo.c
+++ b/fs/proc/meminfo.c
@@ -39,6 +39,7 @@ static int meminfo_proc_show(struct seq_file *m, void *v)
 	unsigned long pages[NR_LRU_LISTS];
 	int lru;
 
+	printk("a\n");
 	si_meminfo(&i);
 	si_swapinfo(&i);
 	committed = percpu_counter_read_positive(&vm_committed_as);
@@ -151,6 +152,7 @@ static int meminfo_proc_show(struct seq_file *m, void *v)
 
 static int __init proc_meminfo_init(void)
 {
+	printk("a\n");
 	proc_create_single("meminfo", 0, NULL, meminfo_proc_show);
 	return 0;
 }
