diff --git a/kernel/time/timer.c b/kernel/time/timer.c
index 786f8c0..1105697 100644
--- a/kernel/time/timer.c
+++ b/kernel/time/timer.c
@@ -1692,6 +1692,9 @@ static __latent_entropy void run_timer_softirq(struct softirq_action *h)
 {
 	struct timer_base *base = this_cpu_ptr(&timer_bases[BASE_STD]);
 
+	if (!base)
+		printk("kpatch-test: testing __tracepoints section changes\n");
+
 	__run_timers(base);
 	if (IS_ENABLED(CONFIG_NO_HZ_COMMON))
 		__run_timers(this_cpu_ptr(&timer_bases[BASE_DEF]));
