diff --git a/kernel/livepatch/patch.c b/kernel/livepatch/patch.c
index 99cb3ad05..9e3617219 100644
--- a/kernel/livepatch/patch.c
+++ b/kernel/livepatch/patch.c
@@ -75,7 +75,7 @@ static void notrace klp_ftrace_handler(unsigned long ip,
 	 * synchronize_rcu() before the func_stack removal.
 	 */
 	if (WARN_ON_ONCE(!func))
-		goto unlock;
+		return;
 
 	/*
 	 * In the enable path, enforce the order of the ops->func_stack and
