diff -Nupr src.orig/fs/proc/meminfo.c src/fs/proc/meminfo.c
--- src.orig/fs/proc/meminfo.c	2022-06-15 10:04:10.540915757 -0400
+++ src/fs/proc/meminfo.c	2022-06-15 10:04:12.104923244 -0400
@@ -29,6 +29,8 @@ static void show_val_kb(struct seq_file
 	seq_write(m, " kB\n", 4);
 }
 
+static int foo = 5;
+
 static int meminfo_proc_show(struct seq_file *m, void *v)
 {
 	struct sysinfo i;
@@ -145,6 +147,7 @@ static int meminfo_proc_show(struct seq_
 	show_val_kb(m, "CmaFree:        ",
 		    global_zone_page_state(NR_FREE_CMA_PAGES));
 #endif
+	seq_printf(m, "kpatch:         %d\n", foo);
 
 	hugetlb_report_meminfo(m);
 
