diff -Nupr src.orig/mm/mmap.c src/mm/mmap.c
--- src.orig/mm/mmap.c	2020-03-17 01:12:57.337061902 -0400
+++ src/mm/mmap.c	2020-03-17 01:13:16.949555374 -0400
@@ -1679,6 +1679,9 @@ unsigned long mmap_region(struct file *f
 	struct rb_node **rb_link, *rb_parent;
 	unsigned long charged = 0;
 
+	if (!jiffies)
+		printk("kpatch mmap foo\n");
+
 	/* Check against address space limit. */
 	if (!may_expand_vm(mm, vm_flags, len >> PAGE_SHIFT)) {
 		unsigned long nr_pages;
