diff -Nupr src.orig/fs/proc/cmdline.c src/fs/proc/cmdline.c
--- src.orig/fs/proc/cmdline.c	2020-03-17 01:12:55.154895731 -0400
+++ src/fs/proc/cmdline.c	2020-03-17 01:13:40.492348137 -0400
@@ -17,3 +17,10 @@ static int __init proc_cmdline_init(void
 	return 0;
 }
 fs_initcall(proc_cmdline_init);
+
+#include <linux/printk.h>
+void kpatch_print_message(void)
+{
+	if (!jiffies)
+		printk("hello there!\n");
+}
diff -Nupr src.orig/fs/proc/meminfo.c src/fs/proc/meminfo.c
--- src.orig/fs/proc/meminfo.c	2020-03-17 01:12:55.155895808 -0400
+++ src/fs/proc/meminfo.c	2020-03-17 01:13:40.492348137 -0400
@@ -21,6 +21,8 @@
 #include <asm/pgtable.h>
 #include "internal.h"
 
+void kpatch_print_message(void);
+
 void __attribute__((weak)) arch_report_meminfo(struct seq_file *m)
 {
 }
@@ -57,6 +59,7 @@ static int meminfo_proc_show(struct seq_
 	sreclaimable = global_node_page_state(NR_SLAB_RECLAIMABLE);
 	sunreclaim = global_node_page_state(NR_SLAB_UNRECLAIMABLE);
 
+	kpatch_print_message();
 	show_val_kb(m, "MemTotal:       ", i.totalram);
 	show_val_kb(m, "MemFree:        ", i.freeram);
 	show_val_kb(m, "MemAvailable:   ", available);
