diff -Nupr src.orig/fs/proc/meminfo.c src/fs/proc/meminfo.c
--- src.orig/fs/proc/meminfo.c	2016-11-30 19:39:49.317737234 +0000
+++ src/fs/proc/meminfo.c	2016-11-30 19:39:58.934737234 +0000
@@ -23,6 +23,8 @@ void __attribute__((weak)) arch_report_m
 {
 }
 
+static int foo = 5;
+
 static int meminfo_proc_show(struct seq_file *m, void *v)
 {
 	struct sysinfo i;
@@ -112,6 +114,7 @@ static int meminfo_proc_show(struct seq_
 		"CmaTotal:       %8lu kB\n"
 		"CmaFree:        %8lu kB\n"
 #endif
+		"kpatch: %d"
 		,
 		K(i.totalram),
 		K(i.freeram),
@@ -172,6 +175,7 @@ static int meminfo_proc_show(struct seq_
 		, K(totalcma_pages)
 		, K(global_page_state(NR_FREE_CMA_PAGES))
 #endif
+		,foo
 		);
 
 	hugetlb_report_meminfo(m);
