diff -Nupr src.orig/fs/aio.c src/fs/aio.c
--- src.orig/fs/aio.c	2016-11-30 19:39:49.237737234 +0000
+++ src/fs/aio.c	2016-11-30 19:40:22.706737234 +0000
@@ -271,9 +271,16 @@ static int __init aio_setup(void)
 }
 __initcall(aio_setup);
 
+void kpatch_aio_foo(void)
+{
+	if (!jiffies)
+		printk("kpatch aio foo\n");
+}
+
 static void put_aio_ring_file(struct kioctx *ctx)
 {
 	struct file *aio_ring_file = ctx->aio_ring_file;
+	kpatch_aio_foo();
 	if (aio_ring_file) {
 		truncate_setsize(aio_ring_file->f_inode, 0);
 
