diff -Nupr src.orig/fs/proc/meminfo.c src/fs/proc/meminfo.c
--- src.orig/fs/proc/meminfo.c	2016-11-30 19:39:49.317737234 +0000
+++ src/fs/proc/meminfo.c	2016-11-30 19:40:40.686737234 +0000
@@ -32,6 +32,7 @@ static int meminfo_proc_show(struct seq_
 	unsigned long pages[NR_LRU_LISTS];
 	int lru;
 
+	printk("a\n");
 /*
  * display in kilobytes.
  */
@@ -196,6 +197,7 @@ static const struct file_operations memi
 
 static int __init proc_meminfo_init(void)
 {
+	printk("a\n");
 	proc_create("meminfo", 0, NULL, &meminfo_proc_fops);
 	return 0;
 }
