diff -Nupr src.orig/fs/proc/cmdline.c src/fs/proc/cmdline.c
--- src.orig/fs/proc/cmdline.c	2017-02-08 21:31:22.297912856 -0500
+++ src/fs/proc/cmdline.c	2017-02-08 21:32:08.510913975 -0500
@@ -27,3 +27,10 @@ static int __init proc_cmdline_init(void
 	return 0;
 }
 fs_initcall(proc_cmdline_init);
+
+#include <linux/printk.h>
+void kpatch_print_message(void)
+{
+	if (!jiffies)
+		printk("hello there!\n");
+}
diff -Nupr src.orig/fs/proc/meminfo.c src/fs/proc/meminfo.c
--- src.orig/fs/proc/meminfo.c	2017-02-08 21:06:25.943876606 -0500
+++ src/fs/proc/meminfo.c	2017-02-08 21:33:26.498915865 -0500
@@ -19,6 +19,8 @@
 #include <asm/pgtable.h>
 #include "internal.h"
 
+void kpatch_print_message(void);
+
 void __attribute__((weak)) arch_report_meminfo(struct seq_file *m)
 {
 }
@@ -65,6 +67,7 @@ static int meminfo_proc_show(struct seq_
 
 	available = si_mem_available();
 
+	kpatch_print_message();
 	show_val_kb(m, "MemTotal:       ", i.totalram);
 	show_val_kb(m, "MemFree:        ", i.freeram);
 	show_val_kb(m, "MemAvailable:   ", available);
