diff -Nupr src.orig/fs/proc/meminfo.c src/fs/proc/meminfo.c
--- src.orig/fs/proc/meminfo.c	2024-07-12 13:29:00.219637700 -0400
+++ src/fs/proc/meminfo.c	2024-07-12 13:29:01.058638072 -0400
@@ -30,6 +30,8 @@ static void show_val_kb(struct seq_file
 	seq_write(m, " kB\n", 4);
 }
 
+static int foo = 5;
+
 static int meminfo_proc_show(struct seq_file *m, void *v)
 {
 	struct sysinfo i;
@@ -155,6 +157,7 @@ static int meminfo_proc_show(struct seq_
 	show_val_kb(m, "CmaFree:        ",
 		    global_zone_page_state(NR_FREE_CMA_PAGES));
 #endif
+	seq_printf(m, "kpatch:         %d\n", foo);
 
 #ifdef CONFIG_UNACCEPTED_MEMORY
 	show_val_kb(m, "Unaccepted:     ",
