diff -Nupr src.orig/net/ipv6/netfilter.c src/net/ipv6/netfilter.c
--- src.orig/net/ipv6/netfilter.c	2024-07-12 13:29:00.495637823 -0400
+++ src/net/ipv6/netfilter.c	2024-07-12 13:29:08.342641303 -0400
@@ -96,6 +96,8 @@ static int nf_ip6_reroute(struct sk_buff
 	return 0;
 }
 
+#include "kpatch-macros.h"
+
 int __nf_ip6_route(struct net *net, struct dst_entry **dst,
 		   struct flowi *fl, bool strict)
 {
@@ -109,6 +111,9 @@ int __nf_ip6_route(struct net *net, stru
 	struct dst_entry *result;
 	int err;
 
+	if (!jiffies)
+		printk("kpatch nf_ip6_route foo\n");
+
 	result = ip6_route_output(net, sk, &fl->u.ip6);
 	err = result->error;
 	if (err)
