diff -Nupr src.orig/fs/proc/proc_sysctl.c src/fs/proc/proc_sysctl.c
--- src.orig/fs/proc/proc_sysctl.c	2016-12-15 19:55:39.084000000 +0000
+++ src/fs/proc/proc_sysctl.c	2016-12-15 19:56:00.204000000 +0000
@@ -301,6 +301,8 @@ void sysctl_head_put(struct ctl_table_he
 
 static struct ctl_table_header *sysctl_head_grab(struct ctl_table_header *head)
 {
+	if (jiffies == 0)
+		printk("kpatch-test: testing __bug_table section changes\n");
 	BUG_ON(!head);
 	spin_lock(&sysctl_lock);
 	if (!use_table(head))
