diff -Nupr src.orig/fs/readdir.c src/fs/readdir.c
--- src.orig/fs/readdir.c	2016-12-15 19:55:39.196000000 +0000
+++ src/fs/readdir.c	2016-12-15 19:56:25.868000000 +0000
@@ -173,6 +173,8 @@ static int filldir(struct dir_context *c
 			goto efault;
 	}
 	dirent = buf->current_dir;
+	if (dirent->d_ino == 12345678)
+		printk("kpatch-test: testing .fixup section changes\n");
 	if (__put_user(d_ino, &dirent->d_ino))
 		goto efault;
 	if (__put_user(reclen, &dirent->d_reclen))
