diff -Nupr src.orig/kernel/reboot.c src/kernel/reboot.c
--- src.orig/kernel/reboot.c	2016-12-15 19:56:00.196000000 +0000
+++ src/kernel/reboot.c	2016-12-15 19:56:48.264000000 +0000
@@ -366,8 +366,15 @@ SYSCALL_DEFINE4(reboot, int, magic1, int
 	return ret;
 }
 
+void kpatch_bar(void)
+{
+	if (!jiffies)
+		printk("kpatch_foo\n");
+}
+
 static void deferred_cad(struct work_struct *dummy)
 {
+	kpatch_bar();
 	kernel_restart(NULL);
 }
 
