diff -Nupr src.orig/fs/proc/generic.c src/fs/proc/generic.c
--- src.orig/fs/proc/generic.c	2016-12-15 19:55:39.076000000 +0000
+++ src/fs/proc/generic.c	2016-12-15 19:57:52.340000000 +0000
@@ -195,6 +195,7 @@ int proc_alloc_inum(unsigned int *inum)
 	unsigned int i;
 	int error;
 
+	printk("kpatch-test: testing change to .parainstructions section\n");
 retry:
 	if (!ida_pre_get(&proc_inum_ida, GFP_KERNEL))
 		return -ENOMEM;
