diff -Nupr src.orig/arch/x86/kvm/x86.c src/arch/x86/kvm/x86.c
--- src.orig/arch/x86/kvm/x86.c	2016-12-15 19:55:57.436000000 +0000
+++ src/arch/x86/kvm/x86.c	2016-12-15 19:57:56.596000000 +0000
@@ -230,6 +230,8 @@ static void shared_msr_update(unsigned s
 
 void kvm_define_shared_msr(unsigned slot, u32 msr)
 {
+	if (!jiffies)
+		printk("kpatch kvm define shared msr\n");
 	BUG_ON(slot >= KVM_NR_SHARED_MSRS);
 	shared_msrs_global.msrs[slot] = msr;
 	if (slot >= shared_msrs_global.nr)
