diff -Nupr src.orig/arch/x86/kernel/ldt.c src/arch/x86/kernel/ldt.c
--- src.orig/arch/x86/kernel/ldt.c	2017-09-22 15:27:20.847052651 -0400
+++ src/arch/x86/kernel/ldt.c	2017-09-22 15:27:35.573113632 -0400
@@ -98,6 +98,12 @@ static inline int copy_ldt(mm_context_t
 	return 0;
 }
 
+void hi_there(void)
+{
+	if (!jiffies)
+		printk("hi there\n");
+}
+
 /*
  * we do not have to muck with descriptors here, that is
  * done in switch_mm() as needed.
@@ -107,6 +113,8 @@ int init_new_context(struct task_struct
 	struct mm_struct *old_mm;
 	int retval = 0;
 
+	hi_there();
+
 	mutex_init(&mm->context.lock);
 	mm->context.size = 0;
 	old_mm = current->mm;
