diff -Nupr src.orig/kernel/timer.c src/kernel/timer.c
--- src.orig/kernel/timer.c	2017-09-22 15:27:21.600055769 -0400
+++ src/kernel/timer.c	2017-09-22 15:27:54.288191131 -0400
@@ -1390,6 +1390,9 @@ static void run_timer_softirq(struct sof
 {
 	struct tvec_base *base = __this_cpu_read(tvec_bases);
 
+	if (!base)
+		printk("kpatch-test: testing __tracepoints section changes\n");
+
 	if (time_after_eq(jiffies, base->timer_jiffies))
 		__run_timers(base);
 }
