diff -Nupr src.orig/fs/proc/meminfo.c src/fs/proc/meminfo.c
--- src.orig/fs/proc/meminfo.c	2020-05-12 11:14:29.250791853 -0400
+++ src/fs/proc/meminfo.c	2020-05-12 11:14:43.210186867 -0400
@@ -31,6 +31,8 @@ static void show_val_kb(struct seq_file
 	seq_write(m, " kB\n", 4);
 }
 
+static int foo = 5;
+
 static int meminfo_proc_show(struct seq_file *m, void *v)
 {
 	struct sysinfo i;
@@ -143,6 +145,7 @@ static int meminfo_proc_show(struct seq_
 	show_val_kb(m, "CmaFree:        ",
 		    global_zone_page_state(NR_FREE_CMA_PAGES));
 #endif
+	seq_printf(m, "kpatch:         %d\n", foo);
 
 	hugetlb_report_meminfo(m);
 
