diff -Nupr src.orig/fs/proc/generic.c src/fs/proc/generic.c
--- src.orig/fs/proc/generic.c	2020-05-12 11:14:29.250791853 -0400
+++ src/fs/proc/generic.c	2020-05-12 11:15:26.558308207 -0400
@@ -205,6 +205,7 @@ int proc_alloc_inum(unsigned int *inum)
 {
 	int i;
 
+	printk("kpatch-test: testing change to .parainstructions section\n");
 	i = ida_simple_get(&proc_inum_ida, 0, UINT_MAX - PROC_DYNAMIC_FIRST + 1,
 			   GFP_KERNEL);
 	if (i < 0)
