diff -Nupr src.orig/kernel/time/timer.c src/kernel/time/timer.c
--- src.orig/kernel/time/timer.c	2020-05-12 11:14:29.670773651 -0400
+++ src/kernel/time/timer.c	2020-05-12 11:15:32.688042552 -0400
@@ -1696,6 +1696,9 @@ static __latent_entropy void run_timer_s
 {
 	struct timer_base *base = this_cpu_ptr(&timer_bases[BASE_STD]);
 
+	if (!base)
+		printk("kpatch-test: testing __tracepoints section changes\n");
+
 	__run_timers(base);
 	if (IS_ENABLED(CONFIG_NO_HZ_COMMON))
 		__run_timers(this_cpu_ptr(&timer_bases[BASE_DEF]));
