diff -Nupr src.orig/fs/proc/meminfo.c src/fs/proc/meminfo.c
--- src.orig/fs/proc/meminfo.c	2021-01-28 04:47:10.916473090 -0500
+++ src/fs/proc/meminfo.c	2021-01-28 04:47:11.459467821 -0500
@@ -29,6 +29,8 @@ static void show_val_kb(struct seq_file
 	seq_write(m, " kB\n", 4);
 }
 
+static int foo = 5;
+
 static int meminfo_proc_show(struct seq_file *m, void *v)
 {
 	struct sysinfo i;
@@ -139,6 +141,7 @@ static int meminfo_proc_show(struct seq_
 	show_val_kb(m, "FilePmdMapped:  ",
 		    global_node_page_state(NR_FILE_PMDMAPPED) * HPAGE_PMD_NR);
 #endif
+	seq_printf(m, "kpatch: %d\n", foo);
 
 #ifdef CONFIG_CMA
 	show_val_kb(m, "CmaTotal:       ", totalcma_pages);
