diff -Nupr src.orig/fs/nfsd/export.c src/fs/nfsd/export.c
--- src.orig/fs/nfsd/export.c	2021-01-28 04:47:10.905473196 -0500
+++ src/fs/nfsd/export.c	2021-01-28 04:55:55.117394790 -0500
@@ -1234,6 +1234,8 @@ static void exp_flags(struct seq_file *m
 	}
 }
 
+extern char *kpatch_string(void);
+
 static int e_show(struct seq_file *m, void *p)
 {
 	struct cache_head *cp = p;
@@ -1243,6 +1245,7 @@ static int e_show(struct seq_file *m, vo
 	if (p == SEQ_START_TOKEN) {
 		seq_puts(m, "# Version 1.1\n");
 		seq_puts(m, "# Path Client(Flags) # IPs\n");
+		seq_puts(m, kpatch_string());
 		return 0;
 	}
 
diff -Nupr src.orig/net/netlink/af_netlink.c src/net/netlink/af_netlink.c
--- src.orig/net/netlink/af_netlink.c	2021-01-28 04:45:47.818280959 -0500
+++ src/net/netlink/af_netlink.c	2021-01-28 04:55:55.134394639 -0500
@@ -2898,4 +2898,9 @@ panic:
 	panic("netlink_init: Cannot allocate nl_table\n");
 }
 
+char *kpatch_string(void)
+{
+	return "# kpatch\n";
+}
+
 core_initcall(netlink_proto_init);
