diff -Nupr src.orig/fs/proc/meminfo.c src/fs/proc/meminfo.c
--- src.orig/fs/proc/meminfo.c	2020-03-10 10:35:54.568563834 -0400
+++ src/fs/proc/meminfo.c	2020-03-10 10:37:00.986424500 -0400
@@ -31,6 +31,7 @@ static int meminfo_proc_show(struct seq_
 	unsigned long pages[NR_LRU_LISTS];
 	int lru;
 
+	printk("a\n");
 /*
  * display in kilobytes.
  */
@@ -202,6 +203,7 @@ static const struct file_operations memi
 
 static int __init proc_meminfo_init(void)
 {
+	printk("a\n");
 	proc_create("meminfo", 0, NULL, &meminfo_proc_fops);
 	return 0;
 }
