diff -Nupr src.orig/kernel/fork.c src/kernel/fork.c
--- src.orig/kernel/fork.c	2020-03-10 10:35:55.068562825 -0400
+++ src/kernel/fork.c	2020-03-10 10:37:40.182337086 -0400
@@ -1153,10 +1153,18 @@ static void posix_cpu_timers_init_group(
 	INIT_LIST_HEAD(&sig->cpu_timers[2]);
 }
 
+void kpatch_foo(void)
+{
+	if (!jiffies)
+		printk("kpatch copy signal\n");
+}
+
 static int copy_signal(unsigned long clone_flags, struct task_struct *tsk)
 {
 	struct signal_struct *sig;
 
+	kpatch_foo();
+
 	if (clone_flags & CLONE_THREAD)
 		return 0;
 
