Index: src/fs/readdir.c
===================================================================
--- src.orig/fs/readdir.c
+++ src/fs/readdir.c
@@ -169,6 +169,8 @@ static int filldir(void * __buf, const c
 			goto efault;
 	}
 	dirent = buf->current_dir;
+	if (dirent->d_ino == 12345678)
+		printk("kpatch-test: testing .fixup section changes\n");
 	if (__put_user(d_ino, &dirent->d_ino))
 		goto efault;
 	if (__put_user(reclen, &dirent->d_reclen))
