diff -Nupr src.orig/fs/proc/proc_sysctl.c src/fs/proc/proc_sysctl.c
--- src.orig/fs/proc/proc_sysctl.c	2021-04-20 11:04:26.717100594 -0400
+++ src/fs/proc/proc_sysctl.c	2021-04-20 11:04:27.636102900 -0400
@@ -338,6 +338,8 @@ static void start_unregistering(struct c
 
 static struct ctl_table_header *sysctl_head_grab(struct ctl_table_header *head)
 {
+	if (jiffies == 0)
+		printk("kpatch-test: testing __bug_table section changes\n");
 	BUG_ON(!head);
 	spin_lock(&sysctl_lock);
 	if (!use_table(head))
