diff -Nupr src.orig/kernel/time/timekeeping.c src/kernel/time/timekeeping.c
--- src.orig/kernel/time/timekeeping.c	2021-04-20 11:04:27.325102120 -0400
+++ src/kernel/time/timekeeping.c	2021-04-20 11:04:39.253132047 -0400
@@ -1231,6 +1231,9 @@ void do_gettimeofday(struct timeval *tv)
 {
 	struct timespec64 now;
 
+	if (!tv)
+		return;
+
 	getnstimeofday64(&now);
 	tv->tv_sec = now.tv_sec;
 	tv->tv_usec = now.tv_nsec/1000;
