diff -Nupr src.orig/fs/proc/proc_sysctl.c src/fs/proc/proc_sysctl.c
--- src.orig/fs/proc/proc_sysctl.c	2023-07-13 16:51:26.926089292 -0400
+++ src/fs/proc/proc_sysctl.c	2023-07-13 16:51:46.329051130 -0400
@@ -53,6 +53,7 @@ void proc_sys_poll_notify(struct ctl_tab
 	if (!poll)
 		return;
 
+	printk("kpatch-test: testing gcc .isra function name mangling\n");
 	atomic_inc(&poll->event);
 	wake_up_interruptible(&poll->wait);
 }
