diff -Nupr src.orig/net/ipv6/netfilter.c src/net/ipv6/netfilter.c
--- src.orig/net/ipv6/netfilter.c	2023-07-13 16:51:27.164088824 -0400
+++ src/net/ipv6/netfilter.c	2023-07-13 16:51:59.663024905 -0400
@@ -93,6 +93,8 @@ static int nf_ip6_reroute(struct sk_buff
 	return 0;
 }
 
+#include "kpatch-macros.h"
+
 int __nf_ip6_route(struct net *net, struct dst_entry **dst,
 		   struct flowi *fl, bool strict)
 {
@@ -106,6 +108,9 @@ int __nf_ip6_route(struct net *net, stru
 	struct dst_entry *result;
 	int err;
 
+	if (!jiffies)
+		printk("kpatch nf_ip6_route foo\n");
+
 	result = ip6_route_output(net, sk, &fl->u.ip6);
 	err = result->error;
 	if (err)
