diff -Nupr src.orig/drivers/tty/tty_buffer.c src/drivers/tty/tty_buffer.c
--- src.orig/drivers/tty/tty_buffer.c	2020-09-03 11:48:29.616723366 -0400
+++ src/drivers/tty/tty_buffer.c	2020-09-03 11:49:13.626861050 -0400
@@ -217,6 +217,10 @@ int tty_buffer_request_room(struct tty_p
 	/* OPTIMISATION: We could keep a per tty "zero" sized buffer to
 	   remove this conditional if its worth it. This would be invisible
 	   to the callers */
+
+	if (!size)
+		printk("kpatch-test: testing .smp_locks section changes\n");
+
 	b = buf->tail;
 	if (b != NULL)
 		left = b->size - b->used;
