Index: kernel-rhel7/fs/proc/meminfo.c
===================================================================
--- kernel-rhel7.orig/fs/proc/meminfo.c
+++ kernel-rhel7/fs/proc/meminfo.c
@@ -20,6 +20,8 @@ void __attribute__((weak)) arch_report_m
 {
 }
 
+static int foo = 5;
+
 static int meminfo_proc_show(struct seq_file *m, void *v)
 {
 	struct sysinfo i;
@@ -110,6 +112,7 @@ static int meminfo_proc_show(struct seq_
 		"CmaTotal:       %8lu kB\n"
 		"CmaFree:        %8lu kB\n"
 #endif
+		"kpatch: %d"
 		,
 		K(i.totalram),
 		K(i.freeram),
@@ -175,6 +178,7 @@ static int meminfo_proc_show(struct seq_
 		, K(totalcma_pages)
 		, K(global_page_state(NR_FREE_CMA_PAGES))
 #endif
+		,foo
 		);
 
 	hugetlb_report_meminfo(m);
