Index: kernel-rhel7/fs/proc/meminfo.c
===================================================================
--- kernel-rhel7.orig/fs/proc/meminfo.c
+++ kernel-rhel7/fs/proc/meminfo.c
@@ -30,6 +30,7 @@ static int meminfo_proc_show(struct seq_
 	unsigned long pages[NR_LRU_LISTS];
 	int lru;
 
+	printk("a\n");
 /*
  * display in kilobytes.
  */
@@ -199,6 +200,7 @@ static const struct file_operations memi
 
 static int __init proc_meminfo_init(void)
 {
+	printk("a\n");
 	proc_create("meminfo", 0, NULL, &meminfo_proc_fops);
 	return 0;
 }
