diff -Nupr src.orig/kernel/time/timer.c src/kernel/time/timer.c
--- src.orig/kernel/time/timer.c	2022-04-29 15:52:13.591336446 -0400
+++ src/kernel/time/timer.c	2022-04-29 15:53:24.467588674 -0400
@@ -1740,10 +1740,14 @@ static inline void __run_timers(struct t
 /*
  * This function runs timers and the timer-tq in bottom half context.
  */
+__attribute__((optimize("-fno-optimize-sibling-calls")))
 static __latent_entropy void run_timer_softirq(struct softirq_action *h)
 {
 	struct timer_base *base = this_cpu_ptr(&timer_bases[BASE_STD]);
 
+	if (!base)
+		printk("kpatch-test: testing __tracepoints section changes\n");
+
 	__run_timers(base);
 	if (IS_ENABLED(CONFIG_NO_HZ_COMMON))
 		__run_timers(this_cpu_ptr(&timer_bases[BASE_DEF]));
