diff -Nupr src.orig/fs/proc/proc_sysctl.c src/fs/proc/proc_sysctl.c
--- src.orig/fs/proc/proc_sysctl.c	2017-11-17 15:58:41.115211972 -0500
+++ src/fs/proc/proc_sysctl.c	2017-11-17 15:59:05.548211972 -0500
@@ -47,6 +47,7 @@ void proc_sys_poll_notify(struct ctl_tab
 	if (!poll)
 		return;
 
+	printk("kpatch-test: testing gcc .isra function name mangling\n");
 	atomic_inc(&poll->event);
 	wake_up_interruptible(&poll->wait);
 }
