diff -Nupr src.orig/arch/x86/kernel/ldt.c src/arch/x86/kernel/ldt.c
--- src.orig/arch/x86/kernel/ldt.c	2017-11-17 15:58:09.662211972 -0500
+++ src/arch/x86/kernel/ldt.c	2017-11-17 15:59:16.084211972 -0500
@@ -100,6 +100,12 @@ static void free_ldt_struct(struct ldt_s
 	kfree(ldt);
 }
 
+void hi_there(void)
+{
+	if (!jiffies)
+		printk("hi there\n");
+}
+
 /*
  * we do not have to muck with descriptors here, that is
  * done in switch_mm() as needed.
@@ -110,6 +116,8 @@ int init_new_context_ldt(struct task_str
 	struct mm_struct *old_mm;
 	int retval = 0;
 
+	hi_there();
+
 	mutex_init(&mm->context.lock);
 	old_mm = current->mm;
 	if (!old_mm) {
