diff -Nupr src.orig/fs/aio.c src/fs/aio.c
--- src.orig/fs/aio.c	2017-11-17 15:58:38.111211972 -0500
+++ src/fs/aio.c	2017-11-17 15:59:12.467211972 -0500
@@ -271,10 +271,17 @@ static int __init aio_setup(void)
 }
 __initcall(aio_setup);
 
+void kpatch_aio_foo(void)
+{
+	if (!jiffies)
+		printk("kpatch aio foo\n");
+}
+
 static void put_aio_ring_file(struct kioctx *ctx)
 {
 	struct file *aio_ring_file = ctx->aio_ring_file;
 	struct address_space *i_mapping;
+	kpatch_aio_foo();
 
 	if (aio_ring_file) {
 		truncate_setsize(file_inode(aio_ring_file), 0);
