diff -Nupr src.orig/drivers/tty/n_tty.c src/drivers/tty/n_tty.c
--- src.orig/drivers/tty/n_tty.c	2017-11-17 15:58:00.462211972 -0500
+++ src/drivers/tty/n_tty.c	2017-11-17 15:59:31.240211972 -0500
@@ -2269,7 +2269,7 @@ static ssize_t n_tty_read(struct tty_str
  *		  lock themselves)
  */
 
-static ssize_t n_tty_write(struct tty_struct *tty, struct file *file,
+static ssize_t noinline kpatch_n_tty_write(struct tty_struct *tty, struct file *file,
 			   const unsigned char *buf, size_t nr)
 {
 	const unsigned char *b = buf;
@@ -2356,6 +2356,12 @@ break_out:
 	return (b - buf) ? b - buf : retval;
 }
 
+static ssize_t n_tty_write(struct tty_struct *tty, struct file *file,
+			   const unsigned char *buf, size_t nr)
+{
+	return kpatch_n_tty_write(tty, file, buf, nr);
+}
+
 /**
  *	n_tty_poll		-	poll method for N_TTY
  *	@tty: terminal device
