diff -Nupr src.orig/kernel/fork.c src/kernel/fork.c
--- src.orig/kernel/fork.c	2025-07-01 13:11:10.803930072 -0400
+++ src/kernel/fork.c	2025-07-01 13:11:32.727847632 -0400
@@ -1877,10 +1877,18 @@ static void posix_cpu_timers_init_group(
 	posix_cputimers_group_init(pct, cpu_limit);
 }
 
+void kpatch_foo(void)
+{
+	if (!jiffies)
+		printk("kpatch copy signal\n");
+}
+
 static int copy_signal(unsigned long clone_flags, struct task_struct *tsk)
 {
 	struct signal_struct *sig;
 
+	kpatch_foo();
+
 	if (clone_flags & CLONE_THREAD)
 		return 0;
 
