Index: kernel/fs/proc/proc_sysctl.c
===================================================================
--- kernel.orig/fs/proc/proc_sysctl.c
+++ kernel/fs/proc/proc_sysctl.c
@@ -301,6 +301,8 @@ void sysctl_head_put(struct ctl_table_he
 
 static struct ctl_table_header *sysctl_head_grab(struct ctl_table_header *head)
 {
+	if (jiffies == 0)
+		printk("kpatch-test: testing __bug_table section changes\n");
 	BUG_ON(!head);
 	spin_lock(&sysctl_lock);
 	if (!use_table(head))
