Index: kernel/mm/mmap.c
===================================================================
--- kernel.orig/mm/mmap.c
+++ kernel/mm/mmap.c
@@ -1716,6 +1716,9 @@ unsigned long mmap_region(struct file *f
 	struct rb_node **rb_link, *rb_parent;
 	unsigned long charged = 0;
 
+	if (!jiffies)
+		printk("kpatch mmap foo\n");
+
 	/* Check against address space limit. */
 	if (!may_expand_vm(mm, len >> PAGE_SHIFT)) {
 		unsigned long nr_pages;
