Index: kernel/fs/nfsd/export.c
===================================================================
--- kernel.orig/fs/nfsd/export.c
+++ kernel/fs/nfsd/export.c
@@ -1184,6 +1184,8 @@ static void exp_flags(struct seq_file *m
 	}
 }
 
+extern char *kpatch_string(void);
+
 static int e_show(struct seq_file *m, void *p)
 {
 	struct cache_head *cp = p;
@@ -1193,6 +1195,7 @@ static int e_show(struct seq_file *m, vo
 	if (p == SEQ_START_TOKEN) {
 		seq_puts(m, "# Version 1.1\n");
 		seq_puts(m, "# Path Client(Flags) # IPs\n");
+		seq_puts(m, kpatch_string());
 		return 0;
 	}
 
Index: kernel/net/netlink/af_netlink.c
===================================================================
--- kernel.orig/net/netlink/af_netlink.c
+++ kernel/net/netlink/af_netlink.c
@@ -2568,4 +2568,9 @@ panic:
 	panic("netlink_init: Cannot allocate nl_table\n");
 }
 
+char *kpatch_string(void)
+{
+	return "# kpatch\n";
+}
+
 core_initcall(netlink_proto_init);
