Index: kernel/kernel/fork.c
===================================================================
--- kernel.orig/kernel/fork.c
+++ kernel/kernel/fork.c
@@ -1146,10 +1146,18 @@ static void posix_cpu_timers_init_group(
 	INIT_LIST_HEAD(&sig->cpu_timers[2]);
 }
 
+void kpatch_foo(void)
+{
+	if (!jiffies)
+		printk("kpatch copy signal\n");
+}
+
 static int copy_signal(unsigned long clone_flags, struct task_struct *tsk)
 {
 	struct signal_struct *sig;
 
+	kpatch_foo();
+
 	if (clone_flags & CLONE_THREAD)
 		return 0;
 
